packages/caliper-samples/network/fabric-v1.4.1/swarm3org1peercouchdb/README.md

```diff
@@ -0,0 +1,32 @@
+# About the network
+
+This directory contains a sample __Fabric v1.4.1__ network with the following properties.
+
+## Topology
+* The network has 3 participating organizations.
+* The network has 3 orderer nodes in __Kafka mode__.
```

Comment:
Just curious about how you place 4 kafka brokers in the 3 nodes.

Reply:
This configuration itself does not define hosts/nodes where the containers (incl. kafka brokers) will be deployed. You can choose any deployments (e.g., 3 orderers on 3 nodes and 4 kafka brokers on 4 nodes) using placement policy in docker-compose.yaml. Note that in the previous performance test, only 3 orderers and 3 kafka brokers was used and deployed on 3 nodes.